packages/gatsby-theme-docs/src/utils/helpers/index.js

```diff
@@ -0,0 +1,10 @@
+import computePathWithPathPrefixInIndex from './compute-path-with-path-prefix';
+
+export {
+  markdown2React,
+  computeType,
+  generateAnchorString,
+  capitalizeFirstCharacter,
+} from './generic';
+
+export const computePathWithPathPrefix = computePathWithPathPrefixInIndex;
```

Comment:
Suggestion: in general I would move each helper function into its own file and have them all under `utils`, there is no real need to nest them into a subfolder or into the same file.

Reply:
I think that makes sense, but so that the move can be as smooth as possible I suggest this to be done later.

Reply:
Sorry but what's the problem with renaming a couple of files now?